spec/index.html

```diff
@@ -612,6 +623,49 @@ <h3>Version Labels</h3>
         </tr>
       </tbody>
     </table>
+
+    <p>For serializations supporting in-line version announcement,
+      the version announcement SHOULD be made early in the document
+      and certainly before serializing any feature depending on that version.</p>
+
+    <section id="server-considerations" class="informative">
+      <h4>Server Considerations</h4>
+      <p>When serializing a graph or dataset which uses features incompatible
+        with a requested version, servers can consider different alternatives:</p>
+
+      <ol>
+        <li>Eliminate <a>triple terms</a> by using an algorithm such
+          as <em>Unstar</em> (downgrading from version "1.2" to "1.2-basic" or to "1.1").
+          <div class="ednote">Fix this when such an algorithmm is defined.</div></li>
+        <li>Replace <a>directional language-tagged strings</a>
+          by <a>literals</a> with a <a>datatype IRI</a> that
+          uses the <a data-cite="JSON-LD11#the-i18n-namespace">`i18n` namespace</a>, as defined in [[JSON-LD11]]
+          ("1.2" or "1.2-basic" to "1.1").</li>
+        <li>Return `406 "Not Acceptable"`.</li>
+        <li>Ignore the requested version and return a native representation.</li>
+      </ol>
+
+      <p class="note">The suggestion here is to follow the
+        <a href="https://en.wikipedia.org/wiki/Robustness_principle">Robustness principle</a>
+        (also known as Postel's Law):
+        servers should be conservative in what they send, be liberal in what they accept.</p>
+    </section>
+
+    <section id="client-considerations" class="informative">
+      <h4>Client Considerations</h4>
+      <p>If a document has no stated `version` (either by HTTP header or internal),
+        systems can assume the version is `"1.2"`, which is compatible
+        with all RDF versions.</p>
+      <p>When parsing a document with conflicting versions,
+        or when parsing a document using unsupported versions, a parser
+        may:</p>
```

Comment:
```suggestion
        MAY do any of the following:</p>
```

Reply:
This is a non-normative section, so we should avoid MAY. Otherwise, this is fine.

Reply:
```suggestion
        may do any of the following:</p>
```